wee_slack.py

```diff
@@ -2248,6 +2255,11 @@ def process_presence_change(message_json, eventrouter, **kwargs):
         user = kwargs["user"]
```

Comment:
The docs state that we can remove processing of the `user` field:
> Sometimes you'll get a single event for a single user but if you use batch_presence_aware=true, that single user event will be single item in the users array.

Reply:
Should we keep it until it is more stable, or do you want it removed now?

Reply:
Hm, maybe it's best to wait, though I suspect we won't remember to remove it. But we can wait until I get a reply on the bug I submitted at least.

Reply:
Do you have a reference / url for your bug report? I’ll add a TODO comment in the code

Reply:
No, I didn' get any. I don't think it's public, they'll just (hopefully) email me.

Reply:
Ok then just a TODO comment :p